pkg/apiserver/api_installer.go

```diff
@@ -105,7 +106,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 
 	serverVersion := a.group.ServerVersion
```

Comment:
Please add some documentation for the ServerVersion variable, too-- I have no idea what one would expect to find in it.